### DISCUSSION TITLE
I can't deploy remix indie-stack

### DISCUSSION THREAD

**Discussion by ddfonseca** [remix-run/remix#5212] [Q&A] [answered]
I know this might not be a remix issue because it uses fly.io, but since all the github actions and fly.toml are pre-config with this template and I don't have any knowledge in this area, I think this could be the case.

So, after completing every single step describe on README.md, I found this error:

```
==> Verifying app config
--> Verified app config
==> Building image
Searching for image 'registry.fly.io/indie-stack-test:main-67ff31467a7c684a051f44578cc4ae1e20b36c32' remotely...
Error failed to fetch an image or build from source: Could not find image "registry.fly.io/indie-stack-test:main-67ff31467a7c684a051f44578cc4ae1e20b36c32"
```

The name is the same on fly.toml app:

`app = "indie-stack-test"`

## Steps to reproduce
- Select the option indie-stack with `npx create-remix@latest`
- Execute every step on README.md
- See the error on Github Actions.

## Links
- https://github.com/ddfonseca/indie-stack-test
- https://github.com/ddfonseca/indie-stack-test/actions/runs/3981282939/jobs/6824933968

**Comment by kiliman** [comment, marked as answer]
There is an issue with GitHub Actions and Fly.

There is a workaround, as discussed here in the Fly forum.

https://community.fly.io/t/deploying-to-fly-via-github-action-failing/10171/19

**Reply by ddfonseca** [reply]
Ok, thanks! I've search the forum but my keywords didn't match that topic. Thanks!
